crates/scheduler-tee/build.rs

```diff
@@ -0,0 +1,45 @@
+//! This file has been copied from crates/shielder_bindings/build.rs
```

Comment:
This is the third file like this in the same repo, ideally it should be refactored as well, but I can't tell the method to do it. So it is a JIRA task candidate.